Lib/test/test_ctypes/test_functions.py

```diff
@@ -72,7 +72,8 @@ def callback(*args):
 
         self.assertEqual(str(cm.exception),
                          "argument 1: TypeError: one character bytes, "
-                         "bytearray or integer expected")
+                         "bytearray or integer in range(256) expected, "
```

Comment:
I strongly prefer a 'series' comma before the 'or'.  (Also called 'serial', 'Oxford', or 'Harvard' comma.)
```suggestion
                         "bytearray, or integer in range(256) expected, "
```
It is more common in America than Britain (see https://en.wikipedia.org/wiki/Serial_comma) and in America more common than and we generally prefer American over British usages.  I see it as also more consistent with Python syntax.  Since we are editing the line anyway, this is a good time to change.  Ditto for similar examples below, some of which are new rather than existing.